alchemist/alchemist-cognitive-agents/src/main/kotlin/it/unibo/alchemist/characteristics/cognitive/AbstractCognitiveCharacteristic.kt

```diff
@@ -0,0 +1,29 @@
+package it.unibo.alchemist.characteristics.cognitive
+
+abstract class AbstractCognitiveCharacteristic : CognitiveCharacteristic {
+
+    protected var currLevel: Double = 0.0
+
+    override fun level() = currLevel
+
+    abstract fun combinationFunction(): Double
+
+    // TODO: These values must be set inside a configuration file
+    companion object {
+        const val wSensing = 0.5
+        const val wAffectiveBiasing = 1.0
+        const val wPersisting = 0.95
+        const val wAmplifyingFeeling = 1.0
+        const val wInhibitingFeeling = 1.0
+        const val wAmplifyingEvacuation = 1.0
+        const val wInhibitingWalkRand = -1.0
+        const val wAmplifyingIntention = 1.0
+        const val wInhibitingIntention = -1.0
+        const val nMental = 0.9
+        const val nBody = 0.25
+        const val logisticSigma = 20.0
+        const val logisticTau = 0.5
+        const val aLogisticSigma = 2.0
+        const val aLogisticTau = 0.14
```

Comment:
w - n - a - what do they stand for?

Reply:
w stands for greek omega letter, n for greek eta letter and a for advanced, don't know if for w and n there is a better name since they are only parameters

Reply:
Be consistent: if you call an hypotetic "sLogistic" "LogisticSigma", then you should call a "eX" "XEta"

Reply:
ok understood your point